litex_boards/targets/digilent_arty.py

```diff
@@ -93,6 +99,18 @@ def __init__(self, variant="a7-35", toolchain="vivado", sys_clk_freq=int(100e6),
         if with_jtagbone:
             self.add_jtagbone()
 
+        if with_mapped_flash:
+
+            self.submodules.spiflash_phy  = LiteSPIPHY(platform.request("spiflash"), S25FL128S(Codes.READ_1_1_1))
+            self.submodules.spiflash_mmap  = LiteSPI(phy=self.spiflash_phy,
+                clk_freq        = sys_clk_freq,
+                mmap_endianness = self.cpu.endianness)
+            self.add_csr("spiflash_mmap")
```

Comment:
self.add_csr is no longer required (still works but now automatically collected/assigned).

Reply:
removed